### PR TITLE
feat(sdk): add radicalbit_platform_sdk plot module

### PR DESCRIPTION
**Pull request by dvalleri**
Implementations:
- add radicalbit_platform_chart_sdk module with proper poetry file
- add a placeholder chart

### CODE REVIEW

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Review by dtria91 — APPROVED** [review]

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

File: sdk/radicalbit_platform_sdk/charts/chart.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+from .chart_data import ChartData, ConfusionMatrixChartData, NumericalBarChartData
+from .utils import get_chart_header, get_formatted_bucket_data
+
+
+class Chart:
+    def __init__(self) -> None:
+        pass
+
+    def placeholder_chart(self, data: ChartData) -> EChartsRawWidget:
```
Comment (rivamarco) [inline sdk/radicalbit_platform_sdk/charts/chart.py:12]: We can remove this placeholder chart

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Review by rivamarco — APPROVED** [review]

**Pull request merged**